examples/drag_n_drop/Cargo.toml

```diff
@@ -9,8 +9,8 @@ publish = false
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [dependencies]
 bevy_vello = { path = "../../", features = ["svg", "lottie"] }
-bevy = { workspace = true }
+bevy = { workspace = true, features = ["default_font"] }
```

Comment:
The `drag_n_drop` example had a button that says "Open" to open a file. This button wasn't rendering any text due to the missing "default_font" feature. It was previously included by bevy_vello by default, before we feature-gated text.